tripgo.swagger.yaml

```diff
@@ -2506,6 +2723,73 @@ definitions:
       urls:
         - nuremberg-bv-de.hadron.buzzhives.com
 
+  RealtimeSnapshot:
+    description: Real Time Snapshot stats.
```

Comment:
Yeah, I think it's enough to say that it has real-time support and what kind of real-time data it supports.